.github/workflows/draft-release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Draft Release
+
+on:
+  push:
+    # Build and publish artifacts when new tag is created for release
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build Release Artifacts
+        run: IMG="projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:${{ env.GITHUB_REF }}" make build-release-artifacts
```

Comment:
`build-release-artifacts` has 4 targets defined, will the `IMG` variable passed to all of them by default? I think we need to pass it individually, right?

Reply:
This probably you could test in local or have you already tested locally and it works fine?

Reply:
Yes it did work on local test.

```go
$ IMG="projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:v0.1.0-alpha.2" make build-release-artifacts

cp /Users/shamshera/Projects/cluster-api-provider-bringyourownhost/test/e2e/data/infrastructure-provider-byoh/v1beta1/cluster-template.yaml _dist/cluster-template.yaml
sed -i -e 1,20d _dist/cluster-template.yaml
cd config/manager && /Users/shamshera/Projects/cluster-api-provider-bringyourownhost/bin/kustomize 

edit set image gcr.io/k8s-staging-cluster-api/cluster-api-byoh-controller=projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:v0.1.0-alpha.2

/Users/shamshera/Projects/cluster-api-provider-bringyourownhost/bin/kustomize build config/default > _dist/infrastructure-components.yaml
cp metadata.yaml _dist/metadata.yaml
```